handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
 
-### Visibility into Sourcegraph Cloud's attack surface
 
-**Problem:** The Cloud team is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Before we can have [confidence in our security model](#confidence-in-our-security-model), we first need to have visibility into current risks, issues, and threats, so we can proactively plan what work needs to be done (i.e., we don't want to be relying on or waiting for security researchers or customer to report vulnerabilities to us).
 
-**Milestones:**
+### Enable Sourcegraph Cloud
 
-1. ~~All Docker images are scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.21 release)~~
-    1. ~~Three or more vulnerabilities with a high severity, or a CVSS score of at least 4.0 are resolved. (3.21 release)~~
-1. ~~We have resolved container vulnerabilities with a high severity, or a CVSS score of at least 4.0, or that resolution is planned. (3.22 release)~~
-1. Implement centralized storage of all of our existing logs (e.g., application logs, compute infrastructure logs). (3.23 release)
-1. Start collecting audit and access logs (e.g., visibility into both intentional and unintentional logins). This is useful from both a security visibility point of view, as also a requirement for various auditing frameworks (though not currently a target). (3.23 release)
-1. All compute nodes are continuously scanned for known security vulnerabilities and the security team is alerted as vulnerabilities are found. (3.22 release)
-    1. We have resolved compute vulnerabilities with a high severity, or a CVSS score of at least 4.0, or that resolution is planned. (3.23 release)
-1. Normalize log format in our centralized log storage so that it is easier to correlate and search. This is a prerequisite for creating automated alerts from the logs. (3.24 release)
-1. Create alerts and dashboards to automate the process of investigating events of interest (e.g., detect and alert on a spike of failed login attempts to a single account, or across Sourcegraph Cloud as a whole). (3.24 release)
+**Problem:** The Cloud team is working toward [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Before we're comfortable allowing users to index their own private code with Sourcegraph, we need to have confidence in our ability to proactively identify and resolve issues, and we need to have a high degree of confidence in our overall security posture. Note that the current milestones only get us to enabling Sourcegraph cloud for SMBs. This isn't because we don't want to enable cloud for large enterprises as well, but because we don't yet have clarity on enterprise security asks, and what work will be discovered in the process of the currently listed milestones.
```

Comment:
**Problem:** The Cloud team is working towards [private code on Sourcegraph Cloud](../cloud/index.md#private-code-on-sourcegraph-cloud). Prior to allowing users to index their own private code with Sourcegraph, we need confidence that we can proactively identify and resolve issues. We need a high degree of confidence in our security posture. Note that the current milestones only get us to enabling Sourcegraph cloud for SMBs. This isn't because we don't want to enable cloud for large enterprises as well, but because we don't yet have clarity on enterprise security asks, and what work will be discovered in the process of the currently listed milestones.

Reply:
Reworded